content/committees/specification/versioning/index.md

```diff
@@ -0,0 +1,222 @@
+---
+title: "Jakarta EE Specification Versioning, Change, and Deprecation Process 1.0"
+date: "2023-10-10T00:00:00+00:00"
+headline: Jakarta EE Specification Versioning, Change, and Deprecation Process
+aliases:
+  - committees/specification/versioning/
+---
+
+## Content
+
+- [Goals]({{< ref "#goals" >}})
+- [Definitions]({{< ref "#definitions" >}})
+- [Allowed Changes, Deprecation and Versioning]({{< ref "#allowedchanges" >}})
+  - [General Rules]({{< ref "#general" >}}) 
+  - [API]({{< ref "#api" >}})
+  - [Behaviour]({{< ref "#behaviour" >}})
+  - [Component Feature]({{< ref "#component" >}})
+  - [Platform Feature]({{< ref "#platform" >}})
+
+## Goals {#goals}
+
+It is the desire of the Jakarta EE specification projects to be able to make backwards incompatible changes to new 
+versions of Jakarta EE specifications.  The goal of this document is to enable that outcome while also providing 
+guidance so that users can be aware of when backwards incompatible changes have occurred.  The high-level framework in 
+which this document is written:
+
+* Backwards incompatible changes are possible on major versions, but never on minor or patch versions.
+* Users should have at least 1 release to be made aware of a future backwards incompatible change.
+* Requirements for communicating backwards incompatible changes in specification documents and download pages.
+
+It should be noted that this is a departure from the backwards compatibility guarantee previously offered by Java EE.
+
+This process covers the Platform specification, its profiles, and the Component specifications that constitute them. 
+Because of the capability for Component specifications to potentially release at a quicker cadence to the Platform and 
+its profiles, it should be clarified that the Platform specification and its profiles are not allowed to “jump” a 
+Component specification release to introduce breaking changes without first deprecating the feature/behaviour in the 
+Platform specification and its profiles. This is to discourage a Component specification quickly doing two or more 
+back-to-back releases to quickly remove a feature. See the Platform Feature section for further details.
+
+## Definitions {#definitions}
+
+We define four “realms” for changes. For clarity, “changes” here refers to additions (new methods to existing 
+interfaces, and new interfaces), and removals (removing methods from existing interfaces, and removing interfaces 
+entirely). The four realms for changes are:
+
+* API changes – these are changes to the APIs available to applications, vendor implementations, and wrapper 
+implementations. 
+  * As an example, the removal of `isParmetersProvided` from Expression Language.
```

Comment:
```suggestion
  * As an example, the removal of `isParametersProvided` from Expression Language.
```

I found a small typo here

Reply:
That's an intentional typo 👍
I'm giving an example of a misspelt method that was removed.

See the _Removals, Deprecations or Backwards Incompatible Changes_ section of [Expression Language 5.0](https://jakarta.ee/specifications/expression-language/5.0/)

> The deprecated, misspelt method MethodExpression.isParmetersProvided() has been removed.

Reply:
Understood!